dao/src/test/resources/sql/system-test-psql.sql

```diff
@@ -1,5 +1,2 @@
 --PostgreSQL specific truncate to fit constraints
-TRUNCATE TABLE device_credentials, device, device_profile, asset, asset_profile, ota_package, rule_node_state, rule_node, rule_chain, alarm_comment, alarm, entity_alarm;
-
--- Decrease seq_id column to make sure to cover cases of new sequential cycle during the tests
-ALTER SEQUENCE edge_event_seq_id_seq MAXVALUE 256;
```

Comment:
Is this intensional?

Reply:
yes, moved this check to edge black box tests, to verify cloud and edge at the same time:
https://github.com/thingsboard/thingsboard-edge/commit/d874a82c09401f3edb9669221624ce58e115e457